project/src/main/resources/auth/credentials.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Try not to commit the credentials.json. You might have to pull the .gitignore changes down first.

Reply:
ah yeah how would i uncommit credentials

Reply:
nvm ill just delete the file temporarily and push again